packages/loader/container-loader/src/container.ts

```diff
@@ -51,6 +51,7 @@ import {
 	runWithRetry,
```

Comment:
Feedback about overall flow: It's fine for staging, but we need to have very clear flow on what it means to work with partial snapshots. It's less about partial, it's about snapshots that are passed in with all content and not cached by driver.
There should be some indication from this layer (first, later - take into account runtime capabilities somehow) to driver layer that it wants different behavior. And driver to ask SPO differently (as SPO needs to know- can it return partial file or not).

The challenge will be how to resolve chicken-and-egg problem with runtime. I.e. loader needs to load snapshot way before runtime is loaded, so it does not know if runtime can handle this flow. I think we should solve it by simply waiting. I.e. Runtime is almost never behind loader changes. So, if we want one Loop release cycle, and enable this flow after that, then we can make sure that runtime is almost not behind loader changes. And thus this problem does not exist. Runtime would still to deal with old loader/driver though